hw/drivers/ipc_nrf5340/include/ipc_nrf5340/ipc_nrf5340.h

```diff
@@ -131,6 +131,32 @@ uint16_t ipc_nrf5340_available(int channel);
  */
 uint16_t ipc_nrf5340_available_buf(int channel, void **dptr);
 
+/**
+ * Returns number of free space available in IPC ring buffer.
```

Comment:
copy-paste?  same as ipc_nrf5340_data_available_get() and doesn't match implementation